src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -1,17 +1,72 @@
 from __future__ import absolute_import
 
+import math
+from datetime import datetime
 from copy import deepcopy
 
 from rest_framework import status
 from rest_framework.response import Response
 
+from django.utils import timezone
+
 from sentry import features
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.paginator import OffsetPaginator
-from sentry.api.serializers import serialize
+from sentry.api.serializers import serialize, AlertRuleSerializer, CombinedRuleSerializer
 from sentry.incidents.models import AlertRule
-from sentry.incidents.endpoints.serializers import AlertRuleSerializer
+from sentry.models import Rule, RuleStatus
+from sentry.utils.cursors import build_cursor, Cursor
+
+
+class ProjectCombinedRuleIndexEndpoint(ProjectEndpoint):
+    def get(self, request, project):
+        """
+        Fetches alert rules and legacy rules for an organization
+        """
+        if not features.has("organizations:incidents", project.organization, actor=request.user):
+            raise ResourceDoesNotExist
+
+        cursor_string = request.GET.get("cursor", None)
+        page_size = 25
+
+        if cursor_string is None:
+            cursor_string = "0:0:0"
+
+        cursor = Cursor.from_string(cursor_string)
+        cursor_date = datetime.fromtimestamp(float(cursor.value)).replace(tzinfo=timezone.utc)
+
+        alert_rule_queryset = (
+            AlertRule.objects.fetch_for_project(project)
+            .filter(date_added__gte=cursor_date)
```

Comment:
Since we flipped the order, should this be `lte` now?

Reply:
Sorry, I'm not sure what you mean here. What order did we flip?

Reply:
We order descending now, so this will also need to be flipped i think?

Reply:
I think the ordering works properly now - the tests all seem to return the results in the expected order (newest first)